src/actions/arcActions.ts

```diff
@@ -674,19 +665,37 @@ export function createProposal(daoAvatarAddress: string, title: string, descript
       // Go back to home page while action create proposal operation gets carried out
       dispatch(push("/dao/" + daoAvatarAddress));
 
-      await contributionRewardInstance.proposeContributionReward({
-        title,
-        avatar: daoAvatarAddress,
-        beneficiaryAddress,
-        description,
-        ethReward: Util.toWei(ethReward),
-        externalToken: dao.externalTokenAddress ? dao.externalTokenAddress : "",
-        externalTokenReward: dao.externalTokenAddress && externalTokenReward ? Util.toWei(externalTokenReward) : 0,
-        nativeTokenReward: Util.toWei(nativeTokenReward),
-        numberOfPeriods: 1,
-        periodLength: 0,
-        reputationChange: Util.toWei(reputationReward),
-      } as any);
+      const dao = getState().arc.daos[daoAvatarAddress];
+      const contributionRewardInstance = await Arc.ContributionRewardFactory.deployed();
+      const wallet = await getWallet()
+      const contributionReward = new ethers.Contract(contributionRewardInstance.address, contributionRewardArtifacts.abi, wallet)
+
+      if (!beneficiaryAddress.startsWith("0x")) { beneficiaryAddress = "0x" + beneficiaryAddress; }
+      beneficiaryAddress = beneficiaryAddress.toLowerCase();
+
+      const hash = ipfs.hashToHex(ipfsHash)
+
+      const externalTokenAddress = dao.externalTokenAddress ? dao.externalTokenAddress : "0x0000000000000000000000000000000000000000"
+
+      const arg = [
+        daoAvatarAddress,
+        hash,
+        Util.toWei(reputationReward),
+        [
+          Util.toWei(nativeTokenReward),
+          Util.toWei(ethReward),
+          Util.toWei(externalTokenReward),
+          "0",
+          "1"
+        ],
+        externalTokenAddress,
+        beneficiaryAddress
+      ]
+
+      const gasLimit = await contributionReward.estimate.proposeContributionReward(arg[0], arg[1], arg[2], arg[3], arg[4], arg[5])
+      const txReceipt = await contributionReward.proposeContributionReward(arg[0], arg[1], arg[2], arg[3], arg[4], arg[5], { gasLimit })
```

Comment:
We need to send this tx w/out the help of arc.js because of an [unfortunate feature](https://github.com/daostack/arc.js/issues/82)

Reply:
Alchemy has support from arc.js for tracking the tx lifecycle, so I am not sure we can bypass that.  I think best way to address this is probably to change the arc.js behavior.

Reply:
Ok, so @dkent600 , are you going to change this in arc.js?

Reply:
@bohendo @jellegerbrandy  This change has already been released

Reply:
@dkent600 Correct me if I'm wrong but this new release isn't useful here is it?

The [PR that fixes this](https://github.com/daostack/arc.js/pull/382) also upgrades the contract versions to v58 which we don't want yet.

So I don't think there's a version of arc.js available that has this bug fixes and is also using the current contracts (v50 I believe).